020-Connect/010-Postgres/001-direct-access.mdx

```diff
@@ -8,21 +8,23 @@ published: true
 status: beta
 ---
 
-Xata allows you to your database branch via the standard Postgres wire protocol. You can use this connection method to use almost any tool that supports Postgres, including `psql`, `pg_dump / pg_restore`, client drivers in various programming languages, ORMs, BI tools, and more.
+> This is a beta feature that requires a workspace administrator to turn on the `Direct access to Postgres` feature flag within their **workspace settings page**. Once enabled, any _new_ databases created within that workspace will have the option to enable the feature. Existing databases that weren't set up with direct access can not be migrated to allow these features without direct [Xata support](/docs/resources/support) involvement.
```

Comment:
```suggestion
> This is a beta feature that requires a workspace administrator to turn on the `Direct access to Postgres` feature flag within their **workspace settings page**. Once enabled, any _new_ databases created within that workspace will have the option to enable the feature. At this moment, existing databases that weren't set up with direct access can not be migrated to allow these features. For any doubts, please contact [Xata support](/docs/resources/support).
```

Right now we don't have a way to migrate over non-pgroll and pgroll branches, but at some point we will allow that. Also contacting support might not be of help trying to migrate as of now, so limiting the expectations of what support will be able to do.

Reply:
Point taken. I've cleaned up the language (different from yours) and removed the mention of support.